argusAI/utils/event_message_sanitizer.py

```diff
@@ -0,0 +1,197 @@
+import re
+from datetime import datetime
+
+from typing import Pattern, List, Callable
+from uuid import UUID
+
+
+class MessageSanitizer:
```

Comment:
I think it may be a good idea to provide a raw event and a fully sanitized event example here, just to get a quick glance of how events looks pre and post.

Reply:
sanitized events are collected (with run id info) for analysis on host running it
Would be worth to add unit tests for that.
@fruch shall I do it within this PR or followup?